compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/DecomposeAggregateOps.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2024 The IREE Authors
```

Comment:
Can you add a test for this pass with a positive test for `custom_op` and a negative test for another aggregate op (maybe linalg.softmax)